classes/patreon_user_profiles.php

```diff
@@ -55,12 +54,11 @@ function patreon_user_profile_fields( $user ) {
 			<?php
 
 		}
-
 	}
 
 	function save_patreon_user_profile_fields( $user_id ) {
 
-		if ( !current_user_can( 'edit_user', $user_id ) ) {
+		if ( ! current_user_can( 'edit_user', $user_id ) ) {
```

Comment:
do we need this space?

Reply:
Well, its one of the style guidelines at WP.org


> When performing logical comparisons, do it like so:
> if ( ! $foo ) { ...

However, it seems a bit off to me too. We can ditch this guideline if its more preferable.

Reply:
hmmm.. I'll leave this one up to you. It's just a preference thing. your call

Reply:
Reality of the matter is that it feels a bit off for me too. I think we should ignore this guideline.